src/drivers/uavcan/CMakeLists.txt

```diff
@@ -78,7 +78,7 @@ add_definitions(
 	-DUAVCAN_CPP_VERSION=UAVCAN_CPP03
 	-DUAVCAN_DRIVER=uavcan_${UAVCAN_DRIVER}
 	-DUAVCAN_IMPLEMENT_PLACEMENT_NEW=1
-	-DUAVCAN_MEM_POOL_BLOCK_SIZE=48
+	-DUAVCAN_MEM_POOL_BLOCK_SIZE=64
```

Comment:
Just out of curiosity, what defines the needed size?

Reply:
this comment block explains https://github.com/dronecan/libuavcan/blob/9a0fd624c448cad5633720676233f74846387a9f/libuavcan/include/uavcan/build_config.hpp#L205 . I think the original author of UAVCAN has been targeting platforms with smaller pointer width, hence reduced UAVCAN_MEM_POOL_BLOCK_SIZE to 48. But with default size, our PX4 build crashes. Therefore, I increased it to 64 and the build went through.